testutil/testdata/proto.pb.go

```diff
@@ -1608,7 +1608,7 @@ type TestVersion2 struct {
 	//	*TestVersion2_F
 	Sum isTestVersion2_Sum `protobuf_oneof:"sum"`
 	G   *types.Any         `protobuf:"bytes,8,opt,name=g,proto3" json:"g,omitempty"`
-	H   []*TestVersion2    `protobuf:"bytes,9,rep,name=h,proto3" json:"h,omitempty"`
+	H   []*TestVersion1    `protobuf:"bytes,9,rep,name=h,proto3" json:"h,omitempty"`
```

Comment:
@alexanderbez @aaronc im not sure what happened here but i installed tools and ran it again and still getting this. can you guide me to what i need to do?